src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -145,4 +152,21 @@ public static function getSubscribedEvents(): array
      * @return SessionInterface|null A SessionInterface instance or null if no session is available
      */
     abstract protected function getSession();
+
+    /**
+     * Report that the session was unexpectedly used.
```

Comment:
Reports

Reply:
But instead of fixing the typo, I would remove the sentence as the method name is quite clear already.